exercises/spiral-matrix/example.py

```diff
@@ -1,11 +1,11 @@
-def spiral(size):
-    sm = [[0]*size for k in range(size)]
-    i, j, el = 0, -1, 1
+def spiral_matrix(size):
+    matrix = [[0]*size for row in range(size)]
+    i, j, element = 0, -1, 1
     di, dj = [0, 1, 0, -1], [1, 0, -1, 0]
     for x in range(2*size - 1):
         for y in range((2*size - x) // 2):
```

Comment:
This should avoid the `flake8` warning.

```suggestion
        for _ in range((2*size - x) // 2):
```